docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+
+  app:
+    image: ghcr.io/radu-c-martin/maybe:latest
```

Comment:
i assume this will change to whoever owns the image?

Reply:
Unless I'm not thinking about this correctly, we'd set this to Maybe's latest published image.

@Radu-C-Martin I'm guessing this was just here so you could test it on your fork correct?

Reply:
yes, originally I even pre-setup it with the maybe-finance image name, but then you said you'd like people to test it first, which is a fair point 🤷